### DISCUSSION TITLE
How do I download whisper's pre-trained model checkpoints of different sizes

### DISCUSSION THREAD

**Discussion by futian00** [openai/whisper#2418] [Q&A]
All kinds of whisper pre-trained model checkpoints on huggingface are.bin files, but I need the.pt file for the checkpoint, how can I get the.pt file for the whisper pre-trained model checkpoint?

**Comment by phineas-pta** [comment]
https://github.com/openai/whisper/blob/main/whisper/__init__.py

line 18 - 31
